Applications/opensense/test/setup_IMUInverseKinematics_HJC_trial.xml

```diff
@@ -3,6 +3,8 @@
 	<IMUInverseKinematicsTool name="track_HJC_trial">
 		<!--The accuracy of the solution in absolute terms, i.e. the number of significant digits to which the solution can be trusted. Default 1e-6.-->
 		<accuracy>9.9999999999999995e-07</accuracy>
+		<!-- Space fixed Euler angles (XYZ order) from IMU Space to OpenSim. Default to Identity. -->
```

Comment:
FYI, comment here doesn't match new property definition line in IMUInverseKinematicsTool.h.

Reply:
Thanks @tkuchida Indeed I didn't recreate the example file but I went ahead and fixed it by hand. Nice catch